jekyll/_cci2/configuration-reference.md

```diff
@@ -563,11 +572,11 @@ windows.gpu.nvidia.medium<sup>(2)</sup> |   16  | 60  | 1    | Nvidia Tesla T4 |
 {: class="table table-striped"}
 
 ###### Example Usage
+
 ```yaml
 version: 2.1
 orbs:
   win: circleci/windows@2.3.0
-
```

Comment:
I personally prefer more space in the config, makes it easier to read for me. I'm fine with either way, though.